packages/common/src/store/notifications/types.ts

```diff
@@ -196,6 +234,70 @@ export type MilestoneNotification = BaseNotification &
       }
   )
 
+export type MilestoneFollowPushNotification = {
+  // TODO: Not the full structure. Need to verify the fields that come back from identity
```

Comment:
Do we still need all these todos?

Reply:
Yea they're helpful for people walking around this code to know what is complete vs half baked